@xen-orchestra/web/src/components/vm/VmResource.vue

```diff
@@ -0,0 +1,60 @@
+<template>
+  <UiCard class="vm-resource">
+    <UiTitle>
+      {{ $t('resource-management') }}
+    </UiTitle>
+    <template v-for="(value, key) in generalInfo" :key>
+      <VtsQuickInfoRow :label="$t(key)" :value="String(value)" />
+    </template>
+  </UiCard>
+</template>
+
+<script setup lang="ts">
+import type { XoVm } from '@/types/xo/vm.type'
+import VtsQuickInfoRow from '@core/components/quick-info-row/VtsQuickInfoRow.vue'
+import UiCard from '@core/components/ui/card/UiCard.vue'
+import UiTitle from '@core/components/ui/title/UiTitle.vue'
+import { formatSizeRaw } from '@core/utils/size.util'
+import { useI18n } from 'vue-i18n'
+
+type GeneralInfo = {
+  'cpu-mask': string
+  'cpu-weight': number
+  'cpu-cap': number
+  'minimum-cpu-limit': string
+  'maximum-cpu-limit': string
+  'vm-limit-topology': string
+  'minimum-static-memory': string
+  'maximum-static-memory': string
+  'minimum-dynamic-memory': string
+  'maximum-dynamic-memory': string
+  gpus: string
+}
+
+const { vm } = defineProps<{ vm: XoVm }>()
+const { t } = useI18n()
+
+const staticMinMemoryFormated = formatSizeRaw(vm.memory.static[0], 0)
+const staticMaxMemoryFormated = formatSizeRaw(vm.memory.static[1], 0)
+const dynamicMinMemoryFormated = formatSizeRaw(vm.memory.dynamic[0], 0)
+const dynamicMaxMemoryFormated = formatSizeRaw(vm.memory.dynamic[1], 0)
+
+const generalInfo: GeneralInfo = {
+  'cpu-cap': vm.cpuCap ? vm.cpuCap : 0 /* XEN_DEFAULT_CPU_CAP */,
+  'cpu-mask': vm.cpuMask ? vm.cpuMask.join(', ') : '',
+  'cpu-weight': vm.cpuWeight ? vm.cpuWeight : 256 /* XEN_DEFAULT_CPU_WEIGHT */,
+  'minimum-cpu-limit': 'no data',
+  'maximum-cpu-limit': vm.CPUs.max + ' ' + t('cpus'),
+  'vm-limit-topology': vm.coresPerSocket
+    ? t('sockets-with-cores-per-socket', {
+        nSockets: vm.CPUs.max / vm.coresPerSocket,
+        nCores: vm.coresPerSocket,
+      })
+    : '',
+  'minimum-static-memory': `${staticMinMemoryFormated?.value} ${staticMinMemoryFormated?.prefix}`,
+  'maximum-static-memory': `${staticMaxMemoryFormated?.value} ${staticMaxMemoryFormated?.prefix}`,
+  'minimum-dynamic-memory': `${dynamicMinMemoryFormated?.value} ${dynamicMinMemoryFormated?.prefix}`,
+  'maximum-dynamic-memory': `${dynamicMaxMemoryFormated?.value} ${dynamicMaxMemoryFormated?.prefix}`,
+  gpus: vm.VGPUs.join(', '),
+}
```

Comment:
Same as my previous comment, here you have to use a `computed` to make this reactive.
And based on a discussion with @ByScripts, here is the suggestion he made (there are some missing values, it is for the example here):
```suggestion
const resources = computed(() => [
  {
    label: t('cpu-mask'),
    value: vm.cpuMask ? vm.cpuMask.join(', ') : t('none')
  },
  {
    label: t('cpu-weight'),
    value: vm.cpuWeight ? vm.cpuWeight : 256 /* XEN_DEFAULT_CPU_WEIGHT */
  },
  {
    label: t('cpu-cap'),
    value: vm.cpuCap ? vm.cpuCap : 0 /* XEN_DEFAULT_CPU_CAP */
  },
  {
    label: t('minimum-cpu-limit'),
    value: ''
  },
  {
    label: t('maximum-cpu-limit'),
    value: `${vm.CPUs.max} ${t('cpus')}`
  },
  {
    label: t('vm-limit-topology'),
    value: vm.coresPerSocket
      ? t('sockets-with-cores-per-socket', {
        nSockets: vm.CPUs.max / vm.coresPerSocket,
        nCores: vm.coresPerSocket,
      })
      : ''
  },
])
```
And in the template:
```html
<VtsQuickInfoRow v-for="({ label, value }) of resources" :key="label" :label :value />
```